setup.py

```diff
@@ -40,16 +40,15 @@ def main():
             "requests-html",
             "segments>=2.2.0,<3",
             "setuptools",
-            "unicodedataplus",
```

Comment:
Dropping `unicodedataplus` here, since it's used only by the scripts under `data/...`, not by the core `wikipron` package anywhere.